packages/patternlab/styleguide/source/assets/js/modules/responsiveTable.js

```diff
@@ -1,30 +1,23 @@
 export default (function (window, document, $) {
-  // Responsive table HTML structure
-  // <div class="ma__table--responsive">
-  //   <div class="ma__table--responsive__wrapper">
-  //    <table> ... </table>
-  //   </div>
-  // </div>
-
   const $window = $(window);
   let responsiveTables = [];
 
-  // Set the width of each sticky header cell and the width of the sticky
-  // header table.
+  function isHighZoom() {
+    const zoomLevel = Math.round((window.outerWidth / window.innerWidth) * 100);
```

Comment:
The solution works! But I think we should combine both check to account for retina devices 
![Screenshot 2025-03-03 at 10 05 11 AM](https://github.com/user-attachments/assets/c91ca970-2e74-46d3-be79-6531b5f7474d)

Reply:
Safari is not responsive to this method `window.outerWidth / window.innerWidth`

Reply:
ok never mind, safari on desktop and mobile only go up to 300% zoom. You can ignore this. This seems to work fine